controllers/api/v1/token.go

```diff
@@ -34,3 +35,43 @@ func (c *Token) Create() {
 
 	c.serveJSON(&tokenResponse)
 }
+
+func (c *Token) Revoke() {
+	err := c.authenticateClient()
+	if err != nil {
+		c.renderError(err, http.StatusUnauthorized)
+		return
```

Comment:
@junan I saw one of Hoang's PR use `c.StopRun()` inside `c.renderError()`  function, so we can omit `return` after calling `c.renderError()` function 👍
It looks really clean to have it